### ISSUE TITLE
Can I run aerich with FastaApi and Tortoise ?

### ISSUE THREAD

**Issue by piotrrybinski88** [tortoise/aerich#105]
Hi, 
Can I use aerich using FastaPi and Tortoise:
I Init database using:
 tortoise.contrib.fastapi import register_tortoise
If it is possible can you show me example ?
Cheers

**Comment by lqmanh** [comment]
Yes of course. You may check [this app template](https://github.com/lqmanh/fastapi-app) out.

**Comment by waketzheng** [comment]
A single app style demo for fastapi+tortoise+aerich: https://github.com/waketzheng/fastapi-tortoise-poetry-template
